src/apis/fs.cpp

```diff
@@ -35,49 +38,110 @@
 #define W_OK 2
 #endif
 
-#define err(L, idx, err) luaL_error(L, "/%s: %s", fixpath(get_comp(L), lua_tostring(L, idx), false, false).c_str(), err)
+using namespace std::filesystem;
```

Comment:
This is a bit dangerous, as it can mess with variables named `path`, which we use a lot here. I think this is a mistake, as we use `fs::` later on. It's best to remove this and just use `fs::` instead.